src/main/src/main/java/org/geoserver/system/status/OSHISystemInfoCollector.java

```diff
@@ -184,22 +187,11 @@ public OSHISystemInfoCollector() {
                         si = Collections.singletonList(mv);
                         break;
                     }
-                case CPU_LOAD:
```

Comment:
Rather than removing this entirely you could use an `oldTicks` value (like you do below on line 194 for per-CPU) and pass to the `getSystemCpuLoadBetweenTicks()` method.  Alternately you could duplicate the per-cpu logic and sum up the arrays to get a system average.

Reply:
:+1: